static/src/stylesheets/email/_article.scss

```diff
@@ -187,4 +187,82 @@ blockquote {
     padding: 0;
 }
 
+// Supporter Merchandising Module
+
+.subscribe {
+    border-collapse: collapse;
+    background-color: #ffffff;
+    &.sc {
+        width: 100%;
+        max-width: 580px;
+    }
+    .padded-divider {
+        padding: 10px;
+        hr {
+            height: 0px;
+            border: 1px solid #dcdcdc;
```

Comment:
For colours, we try to use variables defined in [_vars.scss](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_vars.scss#L76-L83)